cypress/e2e/shared/annotations.range.test.ts

```diff
@@ -169,9 +169,9 @@ describe('Annotations, but in a different test suite', () => {
               .then(endTimeValue => {
                 expect(endTimeValue).to.equal(startTimeValue)
 
-                const newEndTime = moment(endTimeValue, 'YYYY-MM-DD hh:mm:ss a')
+                const newEndTime = moment(endTimeValue, 'YYYY-MM-DD HH:mm:ss')
                   .add(10, 'minutes')
-                  .format('YYYY-MM-DD hh:mm:ss a')
+                  .format('YYYY-MM-DD HH:mm:ss')
```

Comment:
I tried that actually! I remember you saying that on my previous PR, its having compilation issues. It might be because its inside cypress? I'll send a screenshot 👍

Reply:
<img width="1114" alt="Screen Shot 2021-08-04 at 9 08 00 AM" src="https://user-images.githubusercontent.com/18511823/128215551-0adf09fb-18e1-41eb-ad28-82ceffb30159.png">

Reply:
The tests run locally with that import, but then I tried pushing and CI errored here: https://app.circleci.com/pipelines/github/influxdata/monitor-ci/3564/workflows/222a7b2c-8abe-4279-bbfa-20c8143c2576/jobs/40563